app/models/katello/erratum.rb

```diff
@@ -5,7 +5,8 @@ class Erratum < Katello::Model
     SECURITY = ["security"].freeze
     BUGZILLA = ["bugfix", "recommended"].freeze
     ENHANCEMENT = ["enhancement", "optional"].freeze
-    TYPES = [SECURITY, BUGZILLA, ENHANCEMENT].flatten.freeze
+    OTHER = ["other"].freeze
+    TYPES = [SECURITY, BUGZILLA, ENHANCEMENT, OTHER].flatten.freeze
```

Comment:
My understanding is that EPEL errata can have strings in their metadata that identify the type as `'unspecified'` or `'newpackage'`. (see, for example, https://community.theforeman.org/t/cannot-create-errata-filter-for-unspecified-type/38981) If you intend this new "Other" type to be a catch-all, should probably handle that somehow.

Reply:
Yeah exactly, 'other' is a catch-all. `content_view_erratum_filter.rb` is handling this at the moment (albeit in a rather gross way which sorely needs re-written).